aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -0,0 +1,57 @@
+#include "ATen/ATen.h"
+#include "ATen/cuda/CUDAApplyUtils.cuh"
+
+#include <type_traits>
+
+namespace at { namespace native {
+
+///////////////// bincount /////////////////
+namespace {
+template <typename weights_t, typename integral_t>
```

Comment:
nit: this is reversed from the CPU version.

Reply:
fixed